index.html

```diff
@@ -28,15 +28,12 @@
   <meta name="application-name" content="Quiz App">
 
   <!--
-  Chrome 35 has old :host behavior before :host-context addition.
-  Since Chrome 35 does not have HTMLImports, we can use this as a decent signal
-  to force the ShadowDOM Polyfill.
+  Force the ShadowDOM Polyfill. Doing so removes console exceptions in Chrome,
+  but styling is off on paper-fabs.
   -->
-  <script>
-    if (!('import' in document.createElement('link'))) {
-      var Platform = {flags: {shadow: true}};
-    }
-  </script>
+  <!-- <script>
+    var WebComponents= {flags: {shadow: true}};
+  </script> -->
```

Comment:
Probably, but not using the polyfill forces the errors in Chrome and I don't have the knowledge of the polyfill to know what's going on. I left it and the comment for someone in the future and take a look.